package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "shelley",
+  "version": "0.1.1-alpha.1",
+  "license": "TBC",
+  "description": "UI Component lib... Yes; another one.",
+  "homepage": "https://github.com/action-is-hope/shelley",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/action-is-hope/shelley.git"
+  },
+  "keywords": [
+    "gatsby, typescript"
+  ],
+  "author": "Action is Hope<info@action-is-hope.earth>",
+  "dependencies": {
+    "@stylable/core": "1.0.5",
+    "classnames": "^2.2.6",
+    "gatsby": "^2.18.4",
+    "gatsby-link": "^2.2.25",
+    "gatsby-plugin-react-helmet": "^3.1.16",
+    "gatsby-plugin-stylable": "^1.1.1",
+    "gatsby-plugin-typescript": "^2.1.20",
+    "react-dom": "^16.10.1",
+    "react-helmet": "^5.2.0"
+  },
+  "main": "n/a",
+  "scripts": {
+    "type-check": "tsc",
+    "lint": "eslint --ignore-path .gitignore . --ext ts --ext tsx --ext js --ext jsx",
+    "lint:fix": "npm run lint --fix",
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --parser typescript --trailing-comma es5 --no-semi --single-quote --write \"src/**/*.ts*\"",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "devDependencies": {
+    "@stylable/jest": "1.0.5",
```

Comment:
I don't see any tests, can we throw in a simple example for the button? I know it seems pointless but could as an example. Maybe a scaffold script to build out new components or is that lazy?